js-envs/src/main/scala/org/scalajs/jsenv/RunConfig.scala

```diff
@@ -77,6 +92,12 @@ final class RunConfig private (
   def withLogger(logger: Logger): RunConfig =
     copy(logger = logger)
 
+  def withAdditionalEnv(additionalEnv: Map[String, String]): RunConfig =
+    copy(additionalEnv = additionalEnv)
```

Comment:
I find it confusing that this method is called with*Additional*Env, but it *sets* instead of adding. It's even more confusing that the method below actually *adds* to the map.

Have you considered calling it `env` instead of `additionalEnv`, hence this method would be `withEnv`, and the one below would remain `withAdditionalEnv`?

I understand the env here is *in addition* to the `ExternalJSRun.Config.env`. But that one is anyway already *in addition* to the host process environment vars. It doesn't seem problematic to me to keep `env` at each level.

Reply:
I have renamed it to `env` and removed the helper. While it might be convenient every once in a while, it is likely that at the call-site, the env is just forwarded anyways.

So it is definitely not worth the confusion.